apps/web/app/s/[surveyId]/components/LinkSurvey.tsx

```diff
@@ -211,6 +212,7 @@ export default function LinkSurvey({
           styling={determineStyling()}
           languageCode={languageCode}
           isBrandingEnabled={product.linkSurveyBranding}
+          isInEditor={false}
```

Comment:
Please rename this prop to something like `isResetProgress` as it conveys better about what this does, and is a boolean